src/cookbook/lists/horizontal-list.md

```diff
@@ -47,6 +47,13 @@ ListView(
 
 ## Interactive example
 
+{{site.alert.note}}
+  **Desktop and web only:** This example works in the browser and on the desktop. However, as it scrolls on the horizontal axis (left to right or right to left), hold <kbd>Shift</kbd> when hovering your mouse pointer over the list to scroll it.
```

Comment:
Also:
- Please use line breaks as this line is over 80 chars.
- Use "site.alert.secondary", instead of "site.alert.note". The secondary macro uses "Desktop and web only" as the header.